packaging/docker/publish.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# Cross-arch docker publish helper script
+# Needs docker in version >18.02 due to usage of manifests
+#
+# Copyright: SPDX-License-Identifier: GPL-3.0-or-later
+#
+# Author  : Pavlos Emm. Katsoulakis (paul@netdata.cloud)
+
+set -e
+
+WORKDIR="/tmp/docker" # Temporary folder, removed after script is done
```

Comment:
if this is temporary dir, then why not use `mktemp -d`?

Reply:
1) I didn't put that there, i think you could answer that. Actually i would expect to be asked about the "rm -r" of the folder that it wasn't there before. I was thinking that maybe that folder only needed to be created once for some reason (perhaps as part of a preparatory work)

2) Why go for mktemp?

Reply:
`mktemp` will create truly temporary directory and this way you won't have problems when something else creted such directory.

I don't see any reason for this directory to be persistent. It would be better to allow using default docker config instead (but only if you plan to allow usage of this script from outside of CI).

Reply:
Ok. Adjusting and resolving once i push